src/components/Resources.js

```diff
@@ -7,12 +7,12 @@ import { getDisplayName } from '../services/ColorService';
 import {connect} from "react-redux";
 
 const styles = {
-    allContent: {
+    allContentMobile: {
         height: '100%',
         overflow: 'scroll',
         position: 'relative',
         flexDirection: 'column',
-        backgroundColor: 'white'
+        backgroundColor: '#F6F4F3'
```

Comment:
This is a little nitpicky, but it looks like the background doesn't cover the whole view when everything is collapsed. I struggled with this a bit as well on the list view but I think the solution there works well -- any chance you could apply the same logic here?

Reply:
I have addressed the above comment! It looks like this now:
<img src="https://user-images.githubusercontent.com/42149840/59789815-7999bb80-9283-11e9-8d73-fdfae791630c.png" width=200/>